README.md

```diff
@@ -42,7 +42,11 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> A
+S -> ABA
+A->"r"
+A->"b"
+A->"g"
```

Comment:
# 0

Unfortunately this doesn't work, because the two As in "ABA" (which I assume was supposed to be ASA) can be different colors.